point/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "point",
+  "assets": [
+    {
+      "description": "The native governance and staking token of the Point network",
+      "denom_units": [
+        {
+          "denom": "apoint",
+          "exponent": 0
+        },
+        {
+          "denom": "point",
+          "exponent": 18
+        }
+      ],
+      "base": "apoint",
+      "name": "Point",
+      "display": "point",
+      "symbol": "POINT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/pointnetwork/pointnetwork-dashboard/f143abd8e76a652bca407457d179065381274930/assets/point-logo.svg",
```

Comment:
please use URIs that point to the images that are uploaded to the chain registry. (i.e., https://raw.githubusercontent.com/cosmos/chain-registry/master/point/images/point-logo.(png|svg)

Reply:
@JeremyParish69 thanks will look into this